content/applications/sales/sales/amazon_connector/update.rst

```diff
@@ -2,24 +2,17 @@
 Install the Amazon Connector Authentication Update
 ==================================================
 
-Starting July 2020, the Amazon Connector requires the use of a new authentication
-method that makes it easier for you to set up the Connector.
+Starting July 2022, Amazon change it's API and the Amazon Connector needs to be updated to continue
+to exchange information with Amazon. This also makes it easier for you to set up the Connector.
 
-Prior to the update, Odoo customers who wished to use the Amazon Connector had
-to apply to get developer credentials through Amazon, which was a painful and long
-process that could take weeks. This method of authentication is still technically
-possible, but Amazon will refuse to provide developer credentials for Odoo customers
-from July 2020 onwards and will deactivate such credentials for existing customers.
-
-This documentation will help you install the update and use the new authentication
-flow.
+This documentation will help you install the update and use the new authentication flow.
 
 Note that if your Odoo database was first created after the update was released, the
```

Comment:
Let's turn this into a Note to grab the reader's attention and separate this information from the rest of the instructions on the doc. Also, I'm editing the tense to keep it consistent throughout the sentence.
```suggestion
.. note::
If your Odoo database is created after the update was released, the
```